### PR TITLE
Use official taskcluster config

### PR DESCRIPTION
**Pull request by La0**

### CODE REVIEW

File: backend/code_coverage_backend/backend/__init__.py [hunk 1]
```diff
@@ -20,7 +20,7 @@ def create_app():
     taskcluster.auth()
     taskcluster.load_secrets(
         os.environ.get("TASKCLUSTER_SECRET"),
-        code_coverage_backend.config.PROJECT_NAME,
+        prefixes=["common", "backend", "code-coverage-backend"],
```
Comment (marco-c) [inline backend/code_coverage_backend/backend/__init__.py]: Why do we have `backend` and `code-coverage-backend`?

File: tools/requirements.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 logbook
 structlog
-taskcluster
+taskcluster==24.0.0
```
Comment (marco-c) [inline tools/requirements.txt]: >=?

File: backend/code_coverage_backend/backend/__init__.py (hunk 1 above)
Comment (La0) [reply backend/code_coverage_backend/backend/__init__.py]: I would like to switch to simpler project names in the secret now that we have one secret per "family". With that we can quietly move secrets from `code-coverage-backend` to `backend`

Comment (marco-c) [reply backend/code_coverage_backend/backend/__init__.py]: OK.

**Pull request merged**